src/main/resources/db/migration/V135__amend_facilities_dropdown_options.sql

```diff
@@ -0,0 +1,3 @@
+UPDATE search_facility SET name = 'Loop Hearing' WHERE name LIKE 'Hearing%';
```

Comment:
UPDATE search_facility SET name = 'Loop Hearing' WHERE name = 'Hearing Loop T';

May be more future proof! Although I wouldn't worry too much about the above. Not likely for us to have a Hearing Loop J or Hearing Loop 2 for example